src/node/internal/util.d.ts

```diff
@@ -122,3 +122,4 @@ export function isBoxedPrimitive(
 
 export function getBuiltinModule(id: string): any;
 export function getCallSite(frames: number): Record<string, string>[];
+export function processExitImpl(code: number): void;
```

Comment:
why not?

```suggestion
export function exitProcess(code: number): void;
```

Reply:
I actually prefer `processExitImpl` here since that's pretty much exactly what it is ;-) ... the `process.exit Impl`